languages/Dockerfile.builder.j2

```diff
@@ -20,6 +20,10 @@ RUN /tmp/{{package.script}}
 {% endfor %}
 {% endfor %}
 
+# This is needed for compiled languages, otherwise we get permission errors everywhere
+RUN mkdir /opt/algorithm
+RUN chown $ALGO_UID:$ALGO_UID -R /opt/algorithm
```

Comment:
you don't need -R since you just created an empty directory.  i also think this is specific to c# which ends up creating a `bin` directory specifically whereas something like rust uses `target/release` or whatever.  i think it's okay to make this directory owned by algo but I would just update the comment to say something more specific like `Create the /opt/algorithm directory writeable by algo user so that build scripts can create directories.  For example a compiled language like C# creates a bin directory for the compiled binary`.  I think initially this directory was intentionally not writeable since the build scripts were stored in `/opt/algorithm/bin/build` but since it's moved to `/usr/local/bin/algorithmia-build` i don't think this is a problem

Reply:
Added in latest release